extras/userdoc/md/documentation/part-2-populations-of-neurons.md

```diff
@@ -115,6 +120,43 @@ Note that we are being rather lax with random numbers here. Really we have to
 take more care with them, especially if we are using multiple threads or
 distributing over multiple machines. We will worry about this later.
 
+## Generating populations of neurons with deterministic connections
+
+In the previous handout two neurons were connected using synapse 
+specifications. In this section we extend this example to two populations of 
+ten neurons each.
+
+    import pylab
+    import nest
+    pop1 = nest.Create("iaf_neuron", 10)
+    nest.SetStatus(pop1, {"I_e": 376.0})
+    pop2 = nest.Create("iaf_neuron", 10)
+    multimeter = nest.Create("multimeter", 10)
+    nest.SetStatus(multimeter, {"withtime":True, "record_from":["V_m"]})
+
+If no connectivity pattern is specified, the populations are connected via 
+the default rule, namely `all\_to\_all`. Each neuron of `pop1` is connected 
+to every neuron in `pop2`, resulting in $10^2$ connections.
+
+    nest.Connect(pop1, pop2, syn_spec={"weight":20.0})
+
+Alternatively, the neurans can be connected with the `one\_to\_one`. This 
+means that the first neuron in `pop1` is connected to the first neuron in 
+`pop2`, the second to the second, etc., creating ten connections in total.
+
+    nest.Connect(pop1, pop2, "one_to_one", syn_spec={"weight":20.0, "delay":1.0})
+
+Finally, the multimeters are connected using the default rule
+
+    nest.Connect(multimeter, pop2)
+
+Here we have just used very simple connection schemes. Connectivity patterns 
+requiring the specification of further parameters, such as in-degree or 
+connection probabilities, must be defined in a dictionary containing the key 
+`rule` and the key for parameters associated to the rule. Please see
+<http://www.nest-simulator.org/connection_management/> for an illustrated 
```

Comment:
Since connection_management is referred to several times, maybe it would be a good idea to use `` [Connection management][cm]`` everywhere and set it as ``[cm]: connection_management.md  "Connection management"`` at the end of the file...
This way if we want to change it at some point it will be easier...